lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -216,17 +225,51 @@ std::vector<SetGroupFAR>& SessionState::get_all_far_rules() {
   return FarList_;
 }
 
+SessionFsmState SessionState::get_state() {
+  return curr_state_;
+}
+
+magma::lte::Fsm_state_FsmState SessionState::get_smf_state()
```

Comment:
I think we should a TODO here to indicate that we can eventually migrate to using the proto enum directly to get rid of any translation code.

Reply:
Added a note/comment there.